src/OrchardCore.Modules/OrchardCore.ContentFields/Controllers/UserPickerAdminController.cs

```diff
@@ -48,7 +48,8 @@ public async Task<IActionResult> SearchUsers(string part, string field, string c
             var contentItem = await _contentManager.NewAsync(contentType);
             contentItem.Owner = User.FindFirstValue(ClaimTypes.NameIdentifier);
 
-            if (!await _authorizationService.AuthorizeAsync(User, CommonPermissions.EditContent, contentItem))
+            if (!await _authorizationService.AuthorizeAsync(User, CommonPermissions.EditContent, contentItem) &&
+                !await _authorizationService.AuthorizeAsync(User, CommonPermissions.EditOwnContent, contentItem))
```

Comment:
`EditContent` implies `EditOwnContent`. I don't understand the proposed fix.

Reply:
@Skrypt agreed this isn't it. PR #11878 fixes this issue.